### ISSUE TITLE
Hey guys!

### ISSUE THREAD

**Issue by remotevase** [MBTSites/main#1]
Hey!
I have uploaded the main website. This has a countdown clock until when I upload the completed website, as well as uploading the forums which will have a voice command and shoutbox (live chatroom) intergrated!
@mariobrostech @qwewqa how do you like the idea?

**Comment by qwewqa** [comment]
How exactly do you want to make a chatroom in github

**Comment by remotevase** [comment]
no, it will use vanillaforums with a voice upload addon, as well as a shoutbox addon., and it will use the byethost account that I have set up at
mbtmc.byethost15.com. 
Please note: Nothing is uploaded to the byethost account YET. 
Please also note: I am currently the only one with access to the byethost account
Please finally note: I am the only one who will work on the website. Once the main website is uploaded, then you guys can help
